covalent_azurebatch_plugin/assets/infra/main.tf

```diff
@@ -91,13 +105,15 @@ data "template_file" "executor_config" {
   vars = {
     subscription_id        = var.subscription_id
     tenant_id              = var.tenant_id
-    client_id              = "${azuread_application.batch.application_id}"
-    batch_account_url      = "https://${azurerm_batch_account.covalent.account_endpoint}"
+    client_id              = "${azuread_application.batch.client_id}"
+    client_secret          = "${azuread_service_principal_password.covalent_plugin.value}"
```

Comment:
This is here so that it only outputs this secret to the config file and not as part of the terraform output, otherwise it makes it a bit more difficult to obtain the secret especially when using another abstraction of `covalent deploy` upon this.